students/ChrisH/lesson06/arg_kwarg_lab.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# -----------------------------------------------------------
+# arg_kwarg_lab.py
+#  https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/args_kwargs.html
+# -----------------------------------------------------------
+
+
+def func(fore_color='black', back_color='white', link_color='green', visited_color='gray'):
+    return (fore_color, back_color, link_color, visited_color)
+
+def newfunc(*args, **kwargs):
```

Comment:
give a bit more useful names for these functions rather than just `func` and `new_func`.  That doesn't tell the user of your lib anything useful about what they do.